static/backbone/js/api/views/maps/sidepanel/dataPanel.js

```diff
@@ -4,10 +4,17 @@ define(["marionette",
         "text!" + templateDir + "/sidepanel/dataPanelHeader.html",
         "views/maps/sidepanel/menus/projectsMenu",
         "views/maps/sidepanel/projectTags",
-        "views/maps/sidepanel/itemListManager"
+        "views/maps/sidepanel/itemListManager",
+        "views/maps/sidepanel/shareModal/shareModal"
     ],
-    function (Marionette, _, $, dataPanelHeader, ProjectsMenu, ProjectTags,
-                ItemListManager) {
+    function (Marionette,
+              _,
+              $,
+              dataPanelHeader,
+              ProjectsMenu,
+              ProjectTags,
+              ItemListManager,
+              ShareModal) {
```

Comment:
I'm wondering about the use of a modal here, just in terms of our overall UI scheme. So we have tabs-n-panels, and modals. Tab for configuring layers, modal for configuring snapshots. Just raising this question for further discussion and thought next time we meet.